# HG changeset patch
# User Rik <rik@octave.org>
# Date 1380141864 25200
#      Wed Sep 25 13:44:24 2013 -0700
# Node ID ff5ff67946cb18e32e9e06d050c2b3b9b9d4c8ef
# Parent  326af26556ea2b314a79d77e19feb89d433c9dd3
meshgrid.m: Close @code{} macro in docstring.

* scripts/plot/meshgrid.m: Close @code{} macro in docstring.

diff --git a/scripts/plot/meshgrid.m b/scripts/plot/meshgrid.m
--- a/scripts/plot/meshgrid.m
+++ b/scripts/plot/meshgrid.m
@@ -41,19 +41,19 @@
 ## [@var{X}, @var{Y}] = meshgrid (range, range);  
 ## Z = f (X, Y);
 ## surf (X, Y, Z);
 ## @end example
 ##
 ## Programming Note: @code{meshgrid} is restricted to 2-D or 3-D grid
 ## generation.  The @code{ndgrid} function will generate 1-D through N-D
 ## grids.  However, the functions are not completely equivalent.  If @var{x}
-## is a vector of length M and @var{y} is a vector of length N, then @code
-## meshgrid will produce an output grid which is NxM.  @code{ndgrid} will
-## produce an output which is MxN for the same input.
+## is a vector of length M and @var{y} is a vector of length N, then
+## @code{meshgrid} will produce an output grid which is NxM.  @code{ndgrid}
+## will produce an output which is MxN for the same input.
 ## @seealso{ndgrid, mesh, contour, surf}
 ## @end deftypefn
 
 ## Author: jwe
 
 function [xx, yy, zz] = meshgrid (x, y, z)
 
   if (nargin == 0 || nargin > 3)
